checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/views/PANLengthFilterTest.kt

```diff
@@ -47,29 +47,29 @@ class PANLengthFilterTest {
         assertNull(filteredString)
     }
 
-    @Test
-    fun givenAnIdentifiedCardWithMultipleValidationRulesAndMaximumCardLengthIsEnteredThenShouldNotFilterBasedOnInput() {
-        val nineteenDigitCardInput = "4000000000000000000"
-
-        val cardValidationRule1 = CardValidationRule(null, null, null, 16)
-        val cardValidationRule2 = CardValidationRule(null, null, 20, null)
-        val cardBrand = CardBrand("visa", emptyList(), null, listOf(cardValidationRule1, cardValidationRule2))
-        val validationResult = ValidationResult(partial = true, complete = false)
-        given(cardValidator.validatePAN(nineteenDigitCardInput)).willReturn(Pair(validationResult, cardBrand))
-
-        // Add a '0' to existing string
-        val filteredString = panLengthFilter.filter("0", 0, 1, SpannableStringBuilder(nineteenDigitCardInput), nineteenDigitCardInput.length, nineteenDigitCardInput.length)
-
-        // Verify that no filtering has been done
-        assertNull(filteredString)
-    }
+//    @Test
```

Comment:
should this still be commented out?

Reply:
This test is no longer relevant as we no longer have brands with multiple pan rules